cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js

```diff
@@ -65,4 +53,70 @@ describe('Moderation Tools for Posts', () => {
       });
     });
   });
+
+  it('should show Feature Post button on an unfeatured post for an admin user', () => {
+    cy.fixture('users/adminUser.json').as('adminUser');
+    cy.get('@adminUser').then((user) => {
+      cy.loginAndVisit(user, '/').then(() => {
+        cy.findAllByRole('link', { name: 'Unfeatured article' })
+          .first()
+          .click({ force: true });
```

Comment:
I'm wondering why we need to force click these links - it's usually a code smell that a user would struggle to click the link 🤔  If there's a genuine reason, we should add a comment to the spec to explain why we need it

Reply:
@aitchiss Thank you

Reply:
@aitchiss When I removed `force: true` from the clicks, I got the following error in Cypress:



![Screen Shot 2022-05-23 at 11 48 09 AM](https://user-images.githubusercontent.com/32520970/169869635-19957417-6ab1-4bf7-9541-27aff8fc9f00.png)

Reply:
Hmm yeah so like it says `pointer-events: none` means that a normal user can't click this element, which is why Cypress doesn't want to click it either 😄  

Thinking about it, do we need to do this at all? If we set the slug for the article in seeds, then we could just navigate straight to `/username/slug-for-article` - it would be quicker and save us hacking around this weirdness